ci/gpu/build.sh

```diff
@@ -57,7 +57,7 @@ conda list
 # Fixing Numpy version to avoid RuntimeWarning: numpy.ufunc size changed, may
 # indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
 conda install "cudatoolkit=$CUDA_REL" \
-              "cupy>=6.5.0" "numpy=1.16.4" \
+              "cupy<7.2.0a0" "numpy=1.16.4" \
```

Comment:
Any reason why not `cupy>=6.5.0,<7.2.0a0` Also, we have `!=7.1.0` in some other repos.

Reply:
I was being lazy with edits.  I'll change that now

Reply:
resolved in https://github.com/rapidsai/dask-cuda/pull/239/commits/d33b908c81bc82a8dc536b4c51f05b8bd67ba74c